evap/evaluation/templates/contact_modal.html

```diff
@@ -21,72 +24,48 @@ <h5 class="modal-title" id="{{ modal_id }}Label">{{ title }}</h5>
             <form id="{{ modal_id }}Form">
                 <div class="modal-body">
                     {{ teaser }}
-                    <br /><br />
-                    <div class="mb-3">
-                        <label for="{{ modal_id }}SenderName" class="control-label">{% trans 'Sender' %}:</label>
-                        <input type="text" class="form-control mb-3" id="{{ modal_id }}SenderName" disabled value="{{ user.full_name }}" />
-                    </div>
-                    <div class="mb-3">
-                        <label for="{{ modal_id }}Subject" class="control-label">{% trans 'Subject' %}:</label>
-                        <input type="text" class="form-control" id="{{ modal_id }}Subject" disabled value="{{ title }}" />
-                    </div>
-                    <div class="mb-3">
-                        <label for="{{ modal_id }}MessageText" class="control-label">{% trans 'Message' %}:</label>
-                        <textarea autofocus class="form-control" id="{{ modal_id }}MessageText" style="resize:vertical;min-height:150px;"></textarea>
+                    <br/><br/>
+                    <div class="mb-3 modal-grid">
+                        <label for="{{ modal_id }}SenderName"
+                               class="control-label my-auto">{% trans 'Sender' %}:</label>
+                        {% if modal_id == "feedbackModal" and allow_anonymous_feedback_messages %}
+                            <div class="btn-group w-75" role="group"
+                                 aria-label="Basic radio toggle button group">
+                                <input type="radio" class="btn-check" name="btnradio" id="{{ modal_id }}SenderName"
```

Comment:
I'd rather not use the default `value`s of "on" as I think most people working on evap do not know this. Could we just explicitly specify two `value`s here for the inputs?